package.json

```diff
@@ -37,9 +37,9 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.800.3",
-    "@angular-devkit/build-ng-packagr": "~0.800.3",
-    "@angular/cli": "~8.0.3",
+    "@angular-devkit/build-angular": "0.800.3",
+    "@angular-devkit/build-ng-packagr": "0.800.3",
+    "@angular/cli": "8.0.3",
```

Comment:
I don't recommend ***.0.4** versions of three above packages because they will _break_ the app.
https://stackoverflow.com/a/56755573